sda-svc/templates/backup-deploy.yaml

```
@@ -1,4 +1,4 @@
-{{- if .Values.backup.deploy}}
+{{- if .Values.global.backup.deploy}}
```

i think it is without the `global`
```suggestion
{{- if .Values.backup.deploy}}
```